packages/components/src/components/Link/Link.test.tsx

```diff
@@ -43,57 +42,53 @@ describe('Link', () => {
   const defaultProps: LinkProps = {
```

Comment:
Maybe worth renaming `commonProps` for consistency with other tests?

Reply:
Renamed